### ISSUE TITLE
Support showing hover on keyboard focus in extensions view

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#199566]
@meganrogge cool! I tried some other places and mostly it works.
One place where it does not work is the Extensions View. It uses the custom hover, and it does not show up on keyboard focus move.

_Originally posted by @isidorn in https://github.com/microsoft/vscode/issues/199199#issuecomment-1831534983_
